Exercises-1-Basic/src/exercises/Num2.java

```diff
@@ -0,0 +1,20 @@
+package exercises;
+
+/*
+ * Write a program to calculate the sum of first 10 natural number.
+//1+2+3+4+5+6+7+8+9+10
```

Comment:
```suggestion
1+2+3+4+5+6+7+8+9+10
```

no need to add comment // as you are already in comment block /* */

Reply:
Done.